.pre-commit-config.yaml

```diff
@@ -0,0 +1,126 @@
+# install:
+#  pip3 install pre-commit
+# run:
+#  pre-commit run -a
+# update deps:
+#  pre-commit autoupdate
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      # see https://github.com/pre-commit/pre-commit-hooks
+      - id: check-added-large-files
+        args: ['--maxkb=500']
+      - id: check-ast
+      - id: check-builtin-literals
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-symlinks
+      - id: check-toml
+      - id: check-xml
+      - id: check-yaml
+      - id: debug-statements
+      - id: destroyed-symlinks
+      # - id: detect-aws-credentials
+      - id: detect-private-key
+      # conflict with black below
+      # - id: double-quote-string-fixer
+      - id: end-of-file-fixer
+      - id: file-contents-sorter
+      - id: fix-byte-order-marker
+      - id: fix-encoding-pragma
+        args: ["--remove"]
+      - id: forbid-new-submodules
+      # - id: forbid-submodules
+      #   args: ['--fix=lf']
+      - id: name-tests-test
+      # - id: no-commit-to-branch
+      #   args: [--branch, staging]
+      - id: pretty-format-json
+        args: ["--autofix"]
+      - id: requirements-txt-fixer
+      # - id: sort-simple-yaml
+      - id: trailing-whitespace
+
+  - repo: https://github.com/asottile/add-trailing-comma

```

Comment:
Black does all my comma handling. ;-)

Reply:
I added this for adding training commas in list of arguments, since otherwise black would allow too long lines.